src/Tasks/DeleteAllJobsTask.php

```diff
@@ -2,52 +2,56 @@
 
 namespace Symbiote\QueuedJobs\Tasks;
 
-use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\Director;
 use SilverStripe\Dev\BuildTask;
+use SilverStripe\PolyExecution\PolyOutput;
 use SilverStripe\ORM\DataObject;
 use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 
 /**
  * An administrative task to delete all queued jobs records from the database.
  * Use with caution!
  */
 class DeleteAllJobsTask extends BuildTask
 {
-    /**
-     * @inheritdoc
-     * @return string
-     */
-    public function getTitle()
+    protected static string $commandName = 'delete-queued-jobs';
```

Comment:
```suggestion
    protected static string $commandName = 'DeleteAllJobsTask';
```

Reply:
Discussion about this and other task command names is being held in the main issue.